### ISSUE TITLE
Process method is not executing when i run IsoServer.java class

### ISSUE THREAD

**Issue by TestAutomate** [aswzen/Simple-JPOS-Q2-HTTP-ISO-Server#1]
Hi, can you help me out using iso server on the socket, my client is sending 0800 messages,but server is not doing anything,not even executing process method.

after [32mServer [ISO8583] Ready On localhost:xxxx[0m nothing happens

Thanks

**Comment by aswzen** [comment]
Will check asap...keep in touch

**Comment by TestAutomate** [comment]
ok thank you

**Comment by TestAutomate** [comment]
Hi ,

Did you manage to look into

Thank you

**Comment by aswzen** [comment]
![image](https://user-images.githubusercontent.com/1474916/48425031-88bd0600-e796-11e8-9fc6-ca7905da6926.png)
I can run the server easily.. i think is your ISO client is sending with wrong packager and channel.. you can use my ISO Client here:

```
package com;

import java.io.IOException;

import org.jpos.iso.ISOChannel;
import org.jpos.iso.ISOException;
import org.jpos.iso.ISOMsg;
import org.jpos.iso.ISOPackager;
import org.jpos.iso.ServerChannel;
import org.jpos.iso.channel.ASCIIChannel;
import org.jpos.iso.channel.XMLChannel;
import org.jpos.iso.packager.ISO87APackager;
import org.jpos.iso.packager.XMLPackager;

public class IsoClientTest {
    public static void main(String[] args) throws ISOException, IOException {
        ISOPackager pgk = new ISO87APackager();
	ServerChannel channel = new ASCIIChannel("127.0.0.1", 10012,pgk);
        channel.connect();
        ISOMsg request = new ISOMsg();
        request.setMTI("0200");
        request.set(2, "16");
        request.set(2, "5421287475388412");
        request.set(3, "000000");
        request.set(4, "400.0");
        request.set(7, "0716070815");
        request.set(11, "844515");
        channel.send(request);
        ISOMsg response = channel.receive();

        response.dump(System.out, "response:");
    }
}
```

*please note about PACKAGER and CHANNEL.. must be same with server

**Comment by aswzen** [comment]
so you have to find the correct channel then.. JPOS included with many builtin channel
![image](https://user-images.githubusercontent.com/1474916/48425512-90c97580-e797-11e8-883a-97c1d2da3871.png)

**Comment by TestAutomate** [comment]
Client is sending in EBCDIC format how to parse that

**Comment by TestAutomate** [comment]
![image](https://user-images.githubusercontent.com/33282524/48426270-54c5ee80-e75e-11e8-8a59-201e14014a0d.png)

**Comment by aswzen** [comment]
no..its security things that how to communicate between server and client.. u have to find out what packager that has been used by your client..

**Comment by TestAutomate** [comment]
how do i find channel and packager by seeing client request

**Comment by aswzen** [comment]
based on my experience.... 
we can't :smile:  
if you can find it then congratz, on my last ISO 8583 project there are some FSD and MOU documents between us and 3rd party/vendor, all specifications on communication things provided in that document

**Comment by TestAutomate** [comment]
hmm what to try now,i tried with BCD

now i got below
RAW      : ððÂ 
MTI      : [34m00B?[0m

**Comment by aswzen** [comment]
![image](https://user-images.githubusercontent.com/1474916/48453549-8b474c00-e7e6-11e8-8acc-2465c58498d9.png)
unicode text?

**Comment by TestAutomate** [comment]
I am not sure how to convert this, i am getting like this
